README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+# hw01
+## Autobiography
+My name is Olivia Padilla, and I am a third-year undergraduate studying Public Policy.  I am from a very small town in the rural Ozarks region of southwest Missouri. Here is a link to my town's webpsite. [link](http://www.cityofcassville.com/). In my free time I like to read, my favorite book is *The Phantom Tollbooth* by Norton Juster. I have three goals for this quarter: 
+1. Practice social distancing. 
+2. Maintain my mental health while doing so. 
+3. Plan an awesome cross-country road-trip for when this is all over.  
+
+## Process
+1. I watched the video Professor Soltoff posted for this assignment. I made notes so that I could follow the correct process by myself.
+2. I forked Homework 01's repository from the class GitHub to my own repository.
+  + I cloned the fork to my computer and copied the repository's URL.
+3. I opened a new project in RStudio Server.
+  + I created a "Version Control Project" with Git.
+  + I used the repository's URL to create this project. 
+  + It took me a little bit to understand that the directory I was using to save this project was located within the university's server and not locally within my own computer. 
```

Comment:
I agree with @ksong814, I really like your idea of nesting an unordered list, within an ordered list to describe the process. I also used a list, but using another list inside the list makes it really easy to read and follow. I'll try to incorporate that as well.